code/__DEFINES/mob.dm

```diff
@@ -21,6 +21,8 @@
 #define AGE_MIN 17			//youngest a character can be
 #define AGE_MAX 85			//oldest a character can be
 
+#define AGE_MIN_E 0			//youngest a character with no age restriction (i.e. dionae, machine people, and grey) can be
```

Comment:
what even
no

follow bay's example if you are going to do this at all, put a var/min_age and var/max_age on species